lib/devops.ts

```diff
@@ -43,6 +43,33 @@ export abstract class DevOpsBaseClient implements IDevOpsClient {
 
   abstract async fetchPullRequests(): Promise<any>;
 
+  filterPRs(devOpsResponse: DevOpsResponse): DevOpsResponse {
+    if (!this.devOpsTeam.filters) {
+      return devOpsResponse;
+    }
+
+    const result = { ...devOpsResponse };
+    result.value = [];
+
+    for (let PR of devOpsResponse.value) {
+      let filterHit = false;
+      for (let filter of this.devOpsTeam.filters) {
+        let regex = RegExp(filter);
```

Comment:
In `IDevOpsTeam.filters` is already specified to be `RegExp[]` so do we need to wrap this in another `RegExp(...)` here?

Reply:
Good point: I had it coming back as a string in my initial tests, but I guess my debugger was lying to me. Replaced it with a much better implementation.

Reply:
In https://github.com/paxos/motivatron/pull/3/commits/772843581da5f00eda2acd2e2e0d0fa5d0f6770b